src/components/input/input.js

```diff
@@ -777,14 +777,6 @@ function ngMessagesAnimation($q, $animateCss) {
 function ngMessageAnimation($animateCss) {
   return {
     enter: function(element, done) {
-      var messages = getMessagesElement(element);
-
-      // If we have the md-auto-hide class, the md-input-invalid animation will fire, so we can skip
-      if (messages.hasClass('md-auto-hide')) {
-        done();
-        return;
```

Comment:
Sorry, what I meant is if removed the `md-auto-hide`, e.g. by having an `ng-if` or `ng-show`.